game-core/src/main/java/games/strategy/engine/framework/startup/ui/SetupPanel.java

```diff
@@ -132,4 +138,8 @@ void layoutPlayerComponents(final JPanel panel, final List<PlayerSelectorRow> pl
     panel.validate();
     panel.repaint();
   }
+
+  public abstract JComponent getDrawable();
+
+  public abstract boolean showCancelButton();
```

Comment:
It appears all implementations of this method answer the query, "should the cancel button be shown?" rather than reacting to the command, "show the cancel button," so the name is a bit misleading.  I'd recommend renaming the method to reflect that it's a query and not a command.  Some ideas include `isCancelButtonVisible()` and `shouldShowCancelButton()`.